loadsbroker/broker.py

```diff
@@ -208,10 +208,10 @@ def _get_run(self, run_id):
             run = None
         return run, session
 
-    async def _run_complete(self, session, mgr, future):
+    def _run_complete(self, session, mgr, future):
```

Comment:
Yea, it's only called as a callback to the RunManager's future. Why was it ever 'yield future' (before 'await future') was a bug, the future is always finished by the time the callback's triggered